nvflare/private/fed/client/client_engine.py

```diff
@@ -347,22 +347,13 @@ def start_app(
         if not os.path.exists(app_root):
             return f"{ERROR_MSG_PREFIX}: Client app does not exist. Please deploy it before starting client."
 
-        app_custom_folder = os.path.join(app_root, "custom")
```

Comment:
Thanks for the fix.

Where is our custom folder adding logic now?

Reply:
It's in the ProcessJobLauncher.